src/main/java/mate/academy/springboot/swagger/repository/ProductRepository.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springboot.swagger.repository;
+
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    @Query("from Product p where  p.price between ?1 and ?2")
```

Comment:
```suggestion
```
will it work without this line?

Reply:
I guess, you can use here only jpa method builder instead of using native query.

https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.query-creation